driverconfig/config.yaml

```diff
@@ -117,12 +117,18 @@ drivers:
         registrar: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.3.0
   - name: isilon
     configVersions:
-      - configVersion: v2.2.0
+      - configVersion: v2.3.0
         useDefaults: true
         supportedVersions:
           - version: v121
           - version: v122
           - version: v123
+      - configVersion: v2.2.0
+        useDefaults: false
+        supportedVersions:
+          - version: v121
+          - version: v122
+          - version: v123
```

Comment:
Please list the sidecars and the versions that should be used for configversion 2.2.0

Reply:
Added the sidecar versions for 2.2.0 config .